applications/GeoMechanicsApplication/custom_utilities/interface_element_utilities.hpp

```diff
@@ -22,132 +22,172 @@ namespace Kratos
 
 class InterfaceElementUtilities
 {
-
 public:
-
     //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-    static inline void CalculateNuMatrix(BoundedMatrix<double,2,4>& rNu,
-                                         const Matrix& Ncontainer,
-                                         const unsigned int& GPoint)
+    static inline void CalculateNuMatrix(BoundedMatrix<double, 2, 4>& rNu,
+                                         const Matrix&                Ncontainer,
+                                         const unsigned int&          GPoint)
     {
-        //Line_interface_2d_2
-        rNu(0,0) = -Ncontainer(GPoint,0); rNu(0,2) = Ncontainer(GPoint,1);
-        rNu(1,1) = -Ncontainer(GPoint,0); rNu(1,3) = Ncontainer(GPoint,1);
+        // Line_interface_2d_2
+        rNu(0, 0) = -Ncontainer(GPoint, 0);
+        rNu(0, 2) = Ncontainer(GPoint, 1);
+        rNu(1, 1) = -Ncontainer(GPoint, 0);
+        rNu(1, 3) = Ncontainer(GPoint, 1);
     }
 
     //----------------------------------------------------------------------------------------
-    static inline void CalculateNuMatrix(BoundedMatrix<double,2,8>& rNu,
-                                         const Matrix& Ncontainer,
-                                         const unsigned int& GPoint)
+    static inline void CalculateNuMatrix(BoundedMatrix<double, 2, 8>& rNu,
+                                         const Matrix&                Ncontainer,
+                                         const unsigned int&          GPoint)
     {
-        //Quadrilateral_interface_2d_4
-        rNu(0,0) = -Ncontainer(GPoint,0); rNu(0,2) = -Ncontainer(GPoint,1);
-        rNu(1,1) = -Ncontainer(GPoint,0); rNu(1,3) = -Ncontainer(GPoint,1);
-
-        rNu(0,4) =  Ncontainer(GPoint,2); rNu(0,6) =  Ncontainer(GPoint,3);
-        rNu(1,5) =  Ncontainer(GPoint,2); rNu(1,7) =  Ncontainer(GPoint,3);
+        // Quadrilateral_interface_2d_4
+        rNu(0, 0) = -Ncontainer(GPoint, 0);
+        rNu(0, 2) = -Ncontainer(GPoint, 1);
+        rNu(1, 1) = -Ncontainer(GPoint, 0);
+        rNu(1, 3) = -Ncontainer(GPoint, 1);
+
+        rNu(0, 4) = Ncontainer(GPoint, 2);
+        rNu(0, 6) = Ncontainer(GPoint, 3);
+        rNu(1, 5) = Ncontainer(GPoint, 2);
+        rNu(1, 7) = Ncontainer(GPoint, 3);
```

Comment:
I agree it looks weird, is basically sorted by matrix row and then ascending columns apparently. There is probably something to do here, e.g. extracting a for-loop that just loops over the rows and clarify what is exactly happening here (because these functions contain a lot of duplications in themselves), but I would prefer to do that separately, since the formatting commits also end up in the .git-blame-ignore-revs (and these kinds of improvements should end up in the git blame).